### ISSUE TITLE
minDate is wrong when passing momentjs instance

### ISSUE THREAD

**Issue by Andreyco** [zippyui/react-date-picker#41]
When I pass monentjs instance to *minDate*, current date is disabled - not selectable.

```js
<DatePicker
  ...
  dateFormat={FORMAT.DATE_FORMAT}
  minDate={moment()}
  ...
/>
```

If I pass very same format value to momentjs instance, it works.
```js
<DatePicker
  ...
  dateFormat={FORMAT.DATE_FORMAT}
  minDate={moment().format(FORMAT.DATE_FORMAT)}
  ...
/>
````

Is this intended behaviour or relevant issue?

**Comment by radubrehar** [comment]
Will look into it. It's probably a bug. Thanks for reporting!

**Comment by radubrehar** [comment]
We've just fixed that and released 2.1.7 to npm.
